packages/foam-core/package.json

```diff
@@ -12,7 +12,7 @@
     "build": "tsdx build --tsconfig ./tsconfig.build.json",
     "test": "tsdx test",
     "lint": "tsdx lint src test",
-    "watch": "tsdx watch",
```

Comment:
All modules atm have an "API" defined in #144 

I don't have a strong opinion about `start` vs `watch`, but we definitely need to keep the consistency - either via renaming across the board, or "aliasing" across the board

Thoughts?

Reply:
I chose `start` because `tsdx doesn't have a `start` and it's what people look for to start an app. If there's no "watch-less" version, I'd just call it `start`.

Reply:
yeah, sounds good - can you update the PR to make sure that all packages (+ top level `package.json`) has the `start` script?